src/common/Infobar.ts

```diff
@@ -5,6 +5,8 @@ export interface InfobarProps {
   description?: string
   buttonLabel?: string
   buttonUrl?: string
+  secondaryButtonLabel?: string
+  secondaryButtonUrl?: string
```

Comment:
how are we going to avoid prop drilling/build nested components in the future? this way of doing things seems to be limited to _just_ the infobar - note, for example, that this wants a nested button but is accepting props for it rather than accepting a button as a child. 

this might complicate things in the future, especially as we build more complex components. worst case is we end up with like a 30 props component (smth that happens in the wild)

Reply:
yup agreed on the points raised, to be honest I think we need to spend some time to think about this after the prototype is done, when we start shifting towards designing the render engine and CMS portion of things

my initial guess is that we could design the render engine such that schemas would still contain nested props but the render engine would be able to parse the button related fields, construct the sub component (`Button` in this case) and pass them to the `Infobar`

Reply:
If this is something to be decided and this is a temporary component that'll be refactored then sure.